### DISCUSSION TITLE
What is the default username/password if you start locally with -slim tag?

### DISCUSSION THREAD

**Discussion by jzakotnik** [openkfw/TruBudget#1312] [Q&A]
If I run a plain docker TruBudget without provisioning, what are the login credentials? What do you think about mentioning the login in the initial Getting Started Readme? 
Apparently most people get it running, then you are at the frontend and don't know how to login..

**Comment by georgimld** [comment]
the root credentials depend on how you started the api, so the ROOT_SECRET env var in the api tells you what the root passwortd is.

So when you start TruBudget with the [`development/start-dev.sh`](https://github.com/openkfw/TruBudget/tree/main/scripts/development) script, then the [`.env_example`](https://github.com/openkfw/TruBudget/blob/main/scripts/development/.env_example) is used, here the [ROOT_SECRET variable](https://github.com/openkfw/TruBudget/blob/main/scripts/development/.env_example#L94) can be checked. But I agree we should update the documentation to specify this!

**Reply by georgimld** [reply]
This was fixed in PR #1325
